Expecta/Matchers/EXPMatchers+raiseWithReason.m

```diff
@@ -16,15 +16,15 @@
         return expectedExceptionCaught;
     });
     
-    failureMessageForTo(^NSString *{
+    failureMessageForTo(^NSString *(id actual) {
```

Comment:
`actual` is not used in the block. Should it be marked with `__unused`? Same for next block.

Reply:
Done.